tsec-libsodium/src/test/scala/tsec/libsodium/KeyExchangeTest.scala

```diff
@@ -32,9 +32,10 @@ class KeyExchangeTest extends SodiumSpec {
         enc2       <- CryptoSecretBox.encrypt[IO](dec1, serverKey2)
         dec2       <- CryptoSecretBox.decrypt[IO](enc2, clientKey2)
 
-      } yield dec2
+      } yield (dec1, dec2)
 
-      program.unsafeRunSync().toUtf8String mustBe s
+      val (dec1, dec2) = program.unsafeRunSync()
+      (dec1.toUtf8String == dec2.toUtf8String && dec1.toUtf8String == plainText.toUtf8String) mustBe true
```

Comment:
I forget my scalatest syntax, but is 

```scala
dec1.toUtf8String must equal (dec2.toUtf8String)
dec1.toUtf8String must equal (plainText.toUtf8String)
```
possible? (for better error messages than `expected: true, but was: false`

Reply:
:+1: on what @hrhino is saying.

Reply:
:+1: